src/toggl_api_private.cc

```diff
@@ -606,6 +607,14 @@ void timeline_event_view_update_duration(TogglTimelineEventView *event_view, con
     event_view->DurationString = copy_string(toggl::Formatter::FormatDuration(duration, toggl::Format::ImprovedOnlyMinAndSec));
```

Comment:
You're leaking memory here by assigning to DurationString without checking if it already contains a pointer.

Reply:
Make sense. Thank for the hint 👍